pylops/LinearOperator.py

```diff
@@ -11,8 +11,13 @@
 from scipy.sparse.linalg import eigsh as sp_eigsh
 from scipy.sparse.linalg import lobpcg as sp_lobpcg
 from scipy.sparse.linalg import lsqr, spsolve
-from scipy.sparse.linalg.interface import _ProductLinearOperator
 
+# need to check scipy version since the interface submodule changed into
+# _interface from scipy>=1.8.0
+if int(sp.__version__.split(".")[1]) < 8:
```

Comment:
Need to check major as well (`int(sp.__version__.split(".")[0]) <= 1`

Reply:
Oh yeah good point, even though we have a harsh requirement on scipy being >=1.4.. but I guess it’s good just in case scipy goes up to 2?